README.md

```diff
@@ -37,15 +37,28 @@ Note, however, that DNF5 cannot yet interface with the following programming lan
 Installing DNF5
 ---------------
 
-To install the DNF5 package manager, use either of the following commands, depending on your Fedora release:
+DNF5 is available since Fedora 38. To install the DNF5 package manager, use either of the following commands, depending on your Fedora release:
 
 | Fedora release | Command |
 | --------------------------- | ------- |
-| 36 | `sudo dnf copr enable rpmsoftwaremanagement/dnf5-unstable && sudo dnf install dnf5`[^copr] |
-| 37 | `sudo dnf copr enable rpmsoftwaremanagement/dnf5-unstable && sudo dnf install dnf5`[^copr] |
-| 38                 | `sudo dnf install dnf5` |
+| 37 | `sudo dnf copr enable rpmsoftwaremanagement/dnf5-nightly && sudo dnf install dnf5`[^copr] |
```

Comment:
Should this be `rpmsoftwaremanagement/dnf-nightly`?

Reply:
Also the [^copr] foot note points to <https://copr.fedorainfracloud.org/coprs/rpmsoftwaremanagement/dnf5-unstable/>. I think it should point to the same COPR repository as the command highlights.